### ISSUE TITLE
BN layer should followed by activation function, and use_bias should be set to False

### ISSUE THREAD

**Issue by ray1422** [CODEJIN/GST_Tacotron#4]
https://github.com/CODEJIN/GST_Tacotron/blob/141c0ccfcb403d13f65b048e7b5f940321d2e604/Modules/Taco2.py#L32

it should be
```python
self.layer.add(tf.keras.layers.Conv1D(
                filters= filters,
                kernel_size= kernel_size,
                strides= stride,
                padding= 'same',
                activation=None,
                use_bias=False
                ))
self.layer.add(tf.keras.layers.BatchNormalization())
self.layer.add(tf.keras.layers.ReLU())
```

**Comment by CODEJIN** [comment]
Hi,

I understood the order of BN and ReLU. But I could not find about no bias. Could you tell me a reference about no bias? Thank you!

**Comment by CODEJIN** [comment]
Okay. I just found no bias reference at **Ioffe, S., & Szegedy, C. (2015).** 

> Note that, since we normalize Wu+b, the bias b can be ignored since its effect will be canceled by the subsequent mean subtraction (the role of the bias is subsumed by β in Alg. 1). 


I will change soon. Thank you so much!
